jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import json
+from traitlets.config.configurable import Configurable
+from traitlets import Int, Float, Unicode, Bool
+
+
+class Git(Configurable):
+    """
+    Remote repository should be configured by the
+    user in their Jupyter config file. Default remote is 'origin'
+    """
+    remote = Unicode(u'origin', config=True)
+
+    def run(self, *args):
```

Comment:
This needs to somehow get a directory that the user is navigating inside of to pass as the `cwd` keyword argument to the `subprocess.check_output` method.

That could either be an additional argument to this method, or a field of the `Git` class (if you wanted to change this class to no longer be a singleton).

Otherwise, all git commands just run in the same directory where Jupyter was started.